test/dotnet-new3.UnitTests/Approvals/AllWebProjectsWork.CanShowHelp_Mvc.approved.txt

```diff
@@ -1,8 +1,22 @@
 ﻿ASP.NET Core Web App (Model-View-Controller) (C#)
 Author: Microsoft
-Description: A project template for creating an ASP.NET Core application with example ASP.NET Core MVC Views and Controllers. This template can also be used for RESTful HTTP services.
+Description: A project template for creating an ASP.NET Core application with example ASP.NET Core MVC Views and Controllers. This template can also be used 
+for RESTful HTTP services.
 This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore/5.0-third-party-notices for details.
-Options:                                                                                                                                                                                   
+
+Usage:
+  new3 mvc [options] [template options]
```

Comment:
I really like the way the template-specific help is laid out here.  I like the named sections in the example usage, and the symmetry with the section names in the detailed output.

The formatting of the options themselves is also good. Variants, defaults, and parameter names are all clearly distinguishable.